src/bluesky/callbacks/fitting.py

```diff
@@ -166,6 +170,8 @@ def center_of_mass(input, labels=None, index=None):
     [(0.33333333333333331, 1.3333333333333333), (3.5, 2.5)]
     """
     normalizer = np.sum(input, labels, index)
+    if normalizer == 0:
+        return None
```

Comment:
can we do `np.nan` instead?  This should probably also be a tuple of right size for the input?

Reply:
Per later discussion, using x.middle instead.